test/malli/core_test.cljc

```diff
@@ -981,7 +981,34 @@
               {:dispatch :type, :decode/string '(fn [x] (update x :type keyword))}
               [:sized [:map [:type 'keyword?] [:size 'int?]]]
               [:human [:map [:type 'keyword?] [:name 'string?] [:address [:map [:country 'keyword?]]]]]]
-             (m/form schema)))))
+             (m/form schema))))
+
+    (let [schema [:multi {:dispatch first}
```

Comment:
[nit] Could test that dispatching `:m/default` works both with `{:dispatch :type}` and `{:dispatch first}`.